lib/i18n/tasks/command/commands/missing.rb

```diff
@@ -48,7 +48,9 @@ def translate_missing(opt = {})
             pattern_re = i18n.compile_key_pattern(opt[:pattern])
             missing.select_keys! { |full_key, _node| full_key =~ pattern_re }
           end
-          translated = i18n.translate_forest missing, from: opt[:from], backend: opt[:backend].to_sym
+
+          backend = i18n.config.dig(:translation, :backend) || opt[:backend]
```

Comment:
This looks like it ignores the command-line flag if the config setting is present?

Reply:
Yes, I think this should be the other way around and the CLI flag should override, will revise.

Reply:
Ah, I realized why I did this - `opt[:backend]` is always present, because a default is being set to `:google`, so it's not possible to tell here if the CLI flag was provided or it's falling back to the default.

I don't think having a default backend is the best idea; if no backend is specified, getting an error message that requests a Google API Key is strange given the user has not requested to use Google.

I will submit a PR to remove the google default, and then it will be possible to swap the order here. Does that sound good?

Reply:
The backend should default to google for backwards compatibility.

Let's set the argument to `nil`. Then here we can do

```ruby
          backend = opt[:backend] || i18n.translation_config[:backend]
```

And set the default backend here:

https://github.com/glebm/i18n-tasks/blob/9852a3f49661babdba58cec841538e637da9f48e/lib/i18n/tasks/configuration.rb#L65

Reply:
Sorry, I may not be clear on what you want here, but if by set the argument to `nil` you mean this:

```ruby
arg :translation_backend,
            '-b',
            '--backend BACKEND',
            t('i18n_tasks.cmd.args.desc.translation_backend'),
            parser: OptionParsers::Locale::Parser,
            default: nil # SET TO NIL HERE
```            

This doesn't work because the `OptionParsers::Locale::Parser` defaults to `en` when the argument is nil. This is one reason why in #581 I replaced this with the `Enum::Parser` - which I modified to accept an argument to not default to the first option.

You could re-open that PR and I could modify it to fall back to `:google` for backwards compatibility if you like?

Or, if I'm not understanding, can you clarify what you would like done here?

I'm also fine with you modifying this PR to what you want if that's easier.

Reply:
Yeah, makes sense.
Let's do everything in this PR, makes it easier to review and refer to in the release notes.